docs/Agent/user-guide/man_page.md

```diff
@@ -425,48 +416,50 @@ Show this message and exit.
 
 Collects data from the registered tools while running a user-specified action. This can be a specific synthetic benchmark workload, a real workload, or simply a delay to measure system activity.
 
-Here are the steps involved
+Invoking pbench-user-benchmark with a workload generator as an argument will perform the following steps:
 
-- Invoking `pbench-user-benchmark` with your workload generator as an argument: that will start the collection tools on all the hosts
-- Next, it will run your workload generator; when that finishes, it will stop the collection tools on all the hosts
-- Finally, the postprocessing phase will gather the data from all the remote hosts and generates `result.txt` file by running the postprocessing tools on everything 
+- Start the collection tools on all the hosts
+- Execute the workload generator
+- Stop the collection tools on all the hosts
+- Gather the data from all the remote hosts and generates a result.txt file by running the tools' post-processing on the collected data
 
 `<command-to-run>`
 A script, executable, or shell command to run while gathering tool data. Use `--`
 to stop processing of `pbench-user-benchmark` options if your command includes
-options, like `pbench-user-benchmark --config string -- fio --bs 16k`.
+options, like 
+> `pbench-user-benchmark --config string -- fio --bs 16k`
 
 **OPTIONS**  
 `-C`, `--config` PATH  
 Path to the Pbench Agent configuration file.
 This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
 
-`--tool-group` STR
-The tool group to use for the list of tools
+`--tool-group` STR  
+The tool group to use for data collection.
 
 `--iteration-list` STR  
 A file containing a list of iterations to run for the provided script;
-the file should contain one iteration per line. With a leading `#` (hash) character used for comments and blank lines are ignored.
-Each iteration line should use alpha-numeric characters before the first space to name the iteration, with the rest of the line provided as arguments to the script;  
+the file should contain one iteration per line. With a leading hash (`#`) character used for comments and blank lines are ignored.
+Each iteration line should use alpha-numeric characters before the first space to name the iteration, with the rest of the line provided as arguments to the script.    
 _NOTE: --iteration-list is not compatible with --use-tool-triggers_
 
 `--sysinfo` STR[,STR...]  
-comma-separated values of system information to be collected; available: `default` `none` `all` `ara` `block` `insights` `kernel_config` `libvirt` `security_mitigations` `sos` `stockpile` `topology`
+Comma-separated values of system information to be collected; available: `default`,`none`,`all`,`ara`,`block`,`insights`,`kernel_config`,`libvirt`,`security_mitigations`,`sos`,`stockpile`,`topology`
```

Comment:
If you put spaces after the commas, then the line can be broken and wrapped naturally.  🙂